docs/docs/download.md

```diff
@@ -57,8 +57,7 @@ Supported architectures: x86_64 and aarch64
 !!! warning "Accessible Directories"
 
 	When using the flatpak installation method, only the `~/.var/app/app.xemu.xemu/data/xemu/xemu` directory can be written to by xemu. The Hard Disk image must be placed there, for example at `~/.var/app/app.xemu.xemu/data/xemu/xemu/xbox_hdd.qcow2`.
-	
-	If you want to whitelist additional directories, you can [do so with Flatseal](troubleshooting.md#flatpak-release-will-not-load-bios-mcpx-rom-hard-drive).
```

Comment:
What's the rationale for removing the Flatseal mention? Not all of our users are savvy in the command line and it has not caused us any support burden, plus it does not require root (not all steam deck users set root passwords). Is there any reason we can't mention both options?

Reply:
Sorry, both options are fine with me.

Reply:
I restored the flatseal instructions.